jobs/tests/test_utils.py

```diff
@@ -167,6 +167,43 @@
             "fennec-42.0b2.fr.android-arm.apk",
         }
     },
+
+
+    # Fennec ARM
+    {
+        "id": "fennec_2017-05-30-10-01-27_55-0a1_android-api-15_multi",
+        "source": {
+            "product": "fennec",
+        },
+        "target": {
+            "version": "55.0a1",
+            "platform": "android-api-15",
+            "locale": "multi",
+            "channel": "nightly"
+        },
+        "download": {
+            "url": "https://archive.mozilla.org/pub/mobile/nightly/2017/05/2017-05-30-10-01-27-"
+            "mozilla-central-android-api-15-old-id/fennec-55.0a1.multi.android-arm.apk",
+        }
+    },
+
+    # Fennec i386
+    {
+        "id": "fennec_2017-05-30-10-01-27_55-0a1_android-i386_multi",
+        "source": {
+            "product": "fennec",
+        },
+        "target": {
+            "version": "55.0a1",
+            "platform": "android-i386",
+            "locale": "multi",
+            "channel": "nightly"
+        },
+        "download": {
+            "url": "https://archive.mozilla.org/pub/mobile/nightly/2017/05/2017-05-30-10-01-27-"
+            "mozilla-central-android-x86-old-id/fennec-55.0a1.multi.android-i386.apk",
```

Comment:
We'll have conflicts on `id` because there's `https://archive.mozilla.org/pub/mobile/nightly/2017/05/2017-05-30-10-01-27-mozilla-central-android-api-15/fennec-55.0a1.multi.android-arm.apk` and `https://archive.mozilla.org/pub/mobile/nightly/2017/05/2017-05-30-10-01-27-mozilla-central-android-api-15-old-id/fennec-55.0a1.multi.android-arm.apk`